build on window failed due to having a dependency on "quale" crate.

Rust toolchain: msvc-1.40.0
errors : 
```
  Compiling quale v1.0.0
error[E0433]: failed to resolve: could not find `unix` in `os`
 --> C:\Users\hesam\.cargo\registry\src\github.com-1ecc6299db9ec823\quale-1.0.0\lib.rs:3:14
  |
3 | use std::os::unix::fs::PermissionsExt;
  |              ^^^^ could not find `unix` in `os`

error[E0425]: cannot find value `X_OK` in crate `libc`
  --> C:\Users\hesam\.cargo\registry\src\github.com-1ecc6299db9ec823\quale-1.0.0\lib.rs:46:50
46 |         libc::access(file_path.into_raw(), libc::X_OK) == libc::EXIT_SUCCESS
   |                                                  ^^^^ not found in `libc`

error[E0425]: cannot find value `S_IXGRP` in crate `libc`
  --> C:\Users\hesam\.cargo\registry\src\github.com-1ecc6299db9ec823\quale-1.0.0\lib.rs:49:32
   |
49 |         (libc::S_IEXEC | libc::S_IXGRP | libc::S_IXOTH) as u32;
   |                                ^^^^^^^ not found in `libc`

```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,7 @@ CHANGES
 1.1.1
 
 - Add `g:glrnvim_gui=1` to the VIML environment. (#20)
+- Fix windows build by using `which` instead of `quale`. (#19)
 
 1.1.0
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 regex = "1"
-quale = "1.0"
+which = "4.0.2"
 dirs = "1.0"
 tempfile = "3.1"
 # 1.0.98 doesn't compile under Mac

--- a/src/backend/mod.rs
+++ b/src/backend/mod.rs
@@ -50,18 +50,19 @@ fn exe_path(exe_path: &Option<String>, exe_name: &str) -> Result<PathBuf, Glrnvi
 
 #[cfg(not(target_os = "macos"))]
 fn find_executable(exe_name: &str) -> Result<PathBuf, GlrnvimError> {
-    match quale::which(exe_name) {
-        Some(p) => Ok(p),
-        None => Err(GlrnvimError::new(format!(
-            "'{}' executable cannot be found.",
-            exe_name
+    match which::which(exe_name) {
+        Ok(p) => Ok(p),
+        Err(e) => Err(GlrnvimError::new(format!(
+            "'{}' executable cannot be found. {}",
+            exe_name,
+            e
         ))),
     }
 }
 
 #[cfg(target_os = "macos")]
 fn find_executable(exe_name: &str) -> Result<PathBuf, GlrnvimError> {
-    if let Some(p) = quale::which(exe_name) {
+    if let Some(p) = which::which(exe_name) {
         return Ok(p);
     }
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,12 +1,10 @@
 extern crate dirs;
-extern crate quale;
 
 mod backend;
 mod config;
 mod error;
 
 use config::*;
-use quale::which;
 use std::env;
 use std::process::Command;
 
@@ -26,7 +24,7 @@ fn prepare_env() {
 }
 
 fn check_nvim() {
-    if which(NVIM_NAME).is_none() {
+    if which::which(NVIM_NAME).is_err() {
         eprintln!("'{}' executable cannot be found.", NVIM_NAME);
         std::process::exit(-1);
     }

